front/src/modules/pathfinding/utils.ts

```diff
@@ -34,6 +34,21 @@ export const formatSuggestedOperationalPoints = (
     coordinates: getPointCoordinates(geometry, pathLength, op.position),
   }));
 
+export const matchPathStepAndOp = (step: PathStep, op: SuggestedOP) => {
+  if ('operational_point' in step) {
+    return step.operational_point === op.opId;
+  }
+  if ('uic' in step) {
+    return step.uic === op.uic && step.ch === op.ch;
+  }
+  if ('trigram' in step) {
+    return step.trigram === op.trigram && step.ch === op.ch;
```

Comment:
I feel like we should check more thant that. Because of some imports issues, we can sometimes have multiple (2 most of the time) ops with same uic/ch. As it's done in other parts in the app, we could add a check for `kp` as well (between gaia and siph I think, the same op can be present 2 times with same uic/ch but two different `kp`) so we are sure we will edit the good one.

Reply:
That makes sense to me in general. Question: is there a downside to always check the `kp`? Looking at `isVia()`, it seems like we sometimes check `kp` and sometimes not - in which situation do we want to ignore `kp`?

Reply:
Also, should we match the `name`?

Reply:
If I remember correctly (should have better documented this), in TimesStops we prefer to check the isVia with kp because some of ops in the list have same uic/ch/name so name is not relevant to compare them.
In SuggestedViaModal, we checked for the name because we decided that the "duplicated" op should add or removed together. But it seems that now it's not iso with TimeStops anymore and it shouldn't be like that.

Reply:
Alright, added the `kp` matching!